internal/server/handlers/updateMetric.go

```diff
@@ -0,0 +1,74 @@
+package handlers
+
+import (
+	"fmt"
+	"github.com/fasdalf/train-go-musthave-metrics/internal/common/constants"
+	"github.com/fasdalf/train-go-musthave-metrics/internal/common/metricstorage"
+	"net/http"
+	"strconv"
+	"strings"
+)
+
+func NewUpdateMetricHandler(s metricstorage.Storage) http.HandlerFunc {
+	const indexType = 0
+	const indexName = 1
+	const indexValue = 2
```

Comment:
Определение и инициализацию констант лучше вынести за пределы функции хендлера. 

```suggestion
const (
	  indexType = 0
	  indexName = 1
	  indexValue = 2
)
	
func NewUpdateMetricHandler(s metricstorage.Storage) http.HandlerFunc {
```

Reply:
Ещё лучше преобразовать его в хендлер gin и использовать именованные параметры. Но для практики и погружения в стандартный пакет хендлер оставлен с типом из ядра. Константы с такими короткими именами должны же будут вызвать неудобства в других хендлерах из пакета рано или поздно?

Reply:
нет больше констант

Reply:
Пока они используются внутри пакета я не вижу особых проблем, но в целом имена должны быть информативными в том контексте в котором они используются.